git.md

```diff
@@ -195,6 +195,26 @@ Fix nasty little bug ISSUE-51 #resolve #time 2h30m
 * Keep the history *clean* and *simple*. Before pushing any branch to the
   public always ensure that it conforms to the style-guide. If it doesn't:
   squash, rebase, reword commits as necessary.
+  
+* When merging multiple small commits into a single change, make sure to
```

Comment:
I think we shouldn't merge/squash multiple small commits into one if they correspond to different tickets. imo, every commit should correspond to one ticket if possible. We could add summary as shown below in pull request overview instead.

Reply:
We could use approach you suggested  but we should change merge strategy.

Reply:
I'm not convinced that anyone will have the self-discipline to do so. This proposition relaxes git guidelines a bit to compensate for really small changes. Making separate commits in those cases is cumbersome and not really helpful anyway. Right now, for example, we have a rule to *amend* pull request instead of adding "Apply review changes" commit. Do you obey this rule? Adding more and more strict rules is not realistic.

Reply:
I don't think that overview filling is more strict than commit squashing.

To be clear: I am not against you approach, I just think than pr overview is good enough.